src/components/web-monitoring-ui.jsx

```diff
@@ -87,9 +105,11 @@ export default class WebMonitoringUi extends React.Component {
         if (!loggedIn) {
           return Promise.reject(new Error('You must be logged in to view pages'));
         }
+        const query = Object.assign(
+          {include_latest: true},
+          this.state.query
+        );
```

Comment:
No major concerns about reformatting this into multiple lines, but I *do* think it’s helpful to have the blank line between bailout/guard clauses like the `if (!loggedIn) return` above and the rest of the logic.